services/file-retriever/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import 'dotenv/config'
+import express, { Application } from 'express'
+import cors from 'cors'
+import { env } from './utils/env.js'
+import { fileRouter } from './http/controllers/file.js'
+
+const app: Application = express()
+
+app.use(
+  cors({
+    origin: env('CORS_ORIGIN', {
+      defaultValue: '*',
+    }),
+  }),
+)
+
+app.use(fileRouter)
+
+app.listen(Number(env('PORT', { defaultValue: 3000 })), () => {
```

Comment:
This is indexer's port

Reply:
I'll update both to avoid confussion

Reply:
`OBJECT_MAPPING_INDEXER_PORT` and `FILE_RETRIEVER_PORT`